pydicom/dataset.py

```diff
@@ -2023,29 +2058,17 @@ def to_json(self, bulk_data_threshold=1, bulk_data_element_handler=None,
         Examples
         --------
         >>> def my_json_dumps(data):
-        ...     return json.dumps(data, indent=4)
+        ...     return json.dumps(data, indent=4, sort_keys=True)
```

Comment:
I think there needs to be a space before the `>>>` or sphinx shows a warning.

Reply:
If it hasn't warned before...

Reply:
I could be thinking of something else